.babelrc

```diff
@@ -90,6 +90,37 @@
                     }
                 ]
             ]
+        },
+        "watch": {
+            "presets": [
```

Comment:
file changed in other commit

Reply:
Yeah, I'm going to merge the babel upgrade first, then update this PR as needed.